apps/cloud/src/api/user.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Created by acunningham on 22/05/17.
+ */
+import Store from '@raincatcher/store';
+import webapi, {ApiService, StoreApiService, WebApiConfig}  from '@raincatcher/webapi';
+import * as express from 'express';
+
+
+// Define User datatype
+interface User {
+  id: string,
+  username: string,
```

Comment:
@TommyJ1994 I think that we can corelate users using username. Do you see any problems with that aproach?